src/facade/dragonfly_connection.cc

```diff
@@ -36,6 +41,10 @@ ABSL_FLAG(std::string, admin_bind, "",
 ABSL_FLAG(std::uint64_t, request_cache_limit, 1ULL << 26,  // 64MB
           "Amount of memory to use for request cache in bytes - per IO thread.");
 
+ABSL_FLAG(bool, no_tls_on_replica, false,
+          "Enable one way TLS for master only, that is, replica does not use TLS to"
```

Comment:
What do you mean by 'one way TLS'?

Reply:
Meaning only the master uses TLS while the client (the replica) does not. Maybe it's a confusing term and I should better change it